css/ndt-test.css

```diff
@@ -0,0 +1,366 @@
+@font-face {
+  font-family: "League Gothic";
+  src: url("../fonts/league-gothic/league_gothic.otf") format("opentype");
+}
+
+body * {
+  margin: 0;
+  padding: 0;
+}
+
+body {
```

Comment:
I don't think so, since what goes in "body {" applies to the body itself, whereas what goes in "body * {" applies specifically to children of the body element. Though it could be that margin and padding are both inherited by all children anyway?